src/test/ui/associated-type/associated-type-projection-from-multiple-supertraits.stderr

```diff
@@ -1,3 +1,9 @@
+error: equality constraints are not yet supported in where clauses (see #20041)
+  --> $DIR/associated-type-projection-from-multiple-supertraits.rs:34:46
+   |
+LL | fn dent_object_2<COLOR>(c: dyn BoxCar) where <dyn BoxCar as Vehicle>::Color = COLOR {
+   |                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This could be further improved by suggesting alternative approaches and by giving it an error code.